mizar/dp/mizar/workflows/builtins/networkpolicies/networkpolicy_util.py

```diff
@@ -0,0 +1,67 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2020 The Authors.
+
+# Authors: Hong Chang   <@Hong-Chang>
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:The above copyright
+# notice and this permission notice shall be included in all copies or
+# substantial portions of the Software.THE SOFTWARE IS PROVIDED "AS IS",
+# WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED
+# TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE
+# FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR
+# THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+from kubernetes import client
+from mizar.common.kubernetes_util import *
+from mizar.dp.mizar.workflows.builtins.networkpolicies.networkpolicy_data_util import *
+from mizar.dp.mizar.operators.endpoints.endpoints_operator import *
+from mizar.dp.mizar.operators.networkpolicies.networkpolicies_operator import *
+
+endpoint_opr = EndpointOperator()
+networkpolicy_opr = NetworkPolicyOperator()
+
```

Comment:
This should be part of networkpolicies/create.py. please merge with create.py workflow

Reply:
If you are concerned about the testing, mizar already has workflow tests written: https://github.com/CentaurusInfra/mizar/commit/c578503b25fb4e39c9bcc7e1cd10198dfcb57a72
We should be doing the same

Reply:
Fixed